src/app/(proper_react)/(redesign)/(authenticated)/admin/feature-flags/page.tsx

```diff
@@ -114,9 +85,13 @@ export default async function FeatureFlagPage() {
         </div>
       </nav>
       <div className={styles.start}>
-        <h1>All Feature Flags</h1>
+        <h1>Flags added here will override Nimbus features.</h1>
+        <h2>Deleted flags will be controllable by Nimbus.</h2>
+        <br />
+        <h3>All Feature Flags</h3>
         <AllFlagsTable data={featureFlags} />
-        <h1>Add New Feature Flag</h1>
+        <br />
+        <h3>Add New Feature Flag</h3>
```

Comment:
Since we are changing the UI, if it's not too much work, it might be useful to add a table below to show the `deleted/archived feature flags`, since they are soft deleted

Reply:
Done